src/catalog/src/builtin.rs

```diff
@@ -4099,6 +4099,22 @@ WHERE worker_id = 0",
     access: vec![PUBLIC_SELECT],
 });
 
+pub static MZ_MAPPABLE_OBJECTS: LazyLock<BuiltinView> = LazyLock::new(|| {
+    BuiltinView {
+    name: "mz_mappable_objects",
+    schema: MZ_INTROSPECTION_SCHEMA,
+    oid: oid::VIEW_MZ_MAPPABLE_OBJECTS_OID,
+    column_defs: None,
+    sql: "
+SELECT SUBSTRING(name FROM 11) AS name, global_id
+FROM mz_introspection.mz_dataflows md JOIN mz_introspection.mz_dataflow_global_ids mdgi USING (id)
+WHERE name LIKE 'Dataflow: %' AND
```

Comment:
Please leave a comment on the dataflow call to remember we can't change it without changing this line, too.